full_plotter.js

```diff
@@ -0,0 +1,1233 @@
+
```

Comment:
This file was used as a reference and contains the original version, but can probably be removed since it's not referenced at all.